packages/rocketchat-api/server/helpers/parseJsonQuery.js

```diff
@@ -44,7 +44,7 @@ RocketChat.API.helperMethods.set('parseJsonQuery', function _parseJsonQuery() {
 		}
 	}
 
-	let query;
+	let query = {};
```

Comment:
with this change I guess the test on [line 58](https://github.com/RocketChat/Rocket.Chat/pull/11657/files#diff-7cea0c6e8f09f3c8044452a26c50567bR58) isn't required anymore, right?

Reply:
Wrong, since `JSON.parse('false');` can generate `query = false`;